### ISSUE TITLE
User friendly variable setting

### ISSUE THREAD

**Issue by GTcreyon** [Sekanor/SM63PracticeHack#58]

**Comment by KevinKib** [comment]
Something like `set VARIABLE_ALIAS VALUE`?

**Comment by GTcreyon** [comment]
> Something like `set VARIABLE_ALIAS VALUE`?

Yeah! Anything along those lines would work. Maybe `/s` as an alternative - although that might be too silimar to /ss
